assets/scss/settings/vendor/sweetalert2/_settings.scss

```diff
@@ -0,0 +1,64 @@
+// SweetAlert2
+// -----------------------------------------------------------------------------
+
+
+//
+// Purpose: Settings for the vendor popup box replacement, SweetAlert2
+//
+// -----------------------------------------------------------------------------
+
+/*
+ * Font color, size, family
+ */
+$swal-font-body:                                      stencilFontFamily("body-font"), Arial, Helvetica, sans-serif;
+$swal-font-bodySize:                                  remCalc(stencilNumber("fontSize-root", "px") + 2px);
+$swal-font-bodyColor:                                 stencilColor("color-textBase");
+$swal-font-headings:                                  stencilFontFamily("headings-font"), Arial, Helvetica, sans-serif;
+$swal-font-headingsSize:                              remCalc(stencilNumber("fontSize-h1", "px") + 2px);
+$swal-font-headingsColor:                             stencilColor("color-textHeading");
+
+// Modal stylings
+$swal-modal-bgColor:                                  stencilColor("navUser-dropdown-backgroundColor") !important;
+$swal-modal-borderRadius:                             4px;
+
+// Input stylings
+$swal-input-color:                                    stencilColor("input-font-color");
+$swal-input-borderColor:                              stencilColor("input-border-color");
+$swal-input-borderColorActive:                        stencilColor("input-border-color-active");
+
+/*
+ * Buttons
+ * -----------------------------------------------------------------------------
+ * !important usage isn't ideal but this prevents needless
+ * declaration of the button color for each swal() instance.
+ * -----------------------------------------------------------------------------
+ */
+
+// Wrapper
+$swal-button-wrapperMargin:                           25px 0 0;
+
+// Confirm button
+$swal-button-bgColor:                                 stencilColor("button--primary-backgroundColor") !important;
```

Comment:
Do you really need to use `!important` here? If the styles are cascaded properly, you shouldn't need to increase their specificity.